src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1588,14 +1591,17 @@
 
 helps['policy assignment identity assign'] = """
 type: command
-short-summary: Add a system assigned identity to a policy assignment.
+short-summary: Add a system assigned identity or a user assigned identity to a policy assignment.
 examples:
   - name: Add a system assigned managed identity to a policy assignment.
     text: >
         az policy assignment identity assign -g MyResourceGroup -n MyPolicyAssignment
   - name: Add a system assigned managed identity to a policy assignment and grant it the 'Contributor' role for the current resource group.
     text: >
-        az policy assignment identity assign -g MyResourceGroup -n MyPolicyAssignment --role Contributor --identity-scope /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup
+        az policy assignment identity assign --identity [system] -g MyResourceGroup -n MyPolicyAssignment --role Contributor --identity-scope /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup
+  - name: Add a user assigned managed identity to a policy assignment.
```

Comment:
Please refer to guide this doc [managed_identity_command_guideline](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design managed identity related commands and parameters
So we should use `--system-assigned` instead of `--identity [system]`

Reply:
Updated.